app/controllers/tasks.py

```diff
@@ -39,11 +40,16 @@
 
 class Tasks(Base):
 
-    @marshal_with(RETURN_TASK_LIST)
+    @pagination(RETURN_TASK_LIST)
     def get(self):
+        args = get_task.parse_args()
         tasks = Task.query.join(UserAndTaskRelation).filter(
             UserAndTaskRelation.user_id == current_app.user.id)
-        return tasks.all()
+        for key, value in args.items():
+            if not value or not dir(Task):
```

Comment:
use hasattr instead of dir, also why you need that check? you've already defined parser with fields.